pyslackersweb/website/contexts.py

```diff
@@ -1,38 +1,51 @@
 from datetime import datetime, timedelta
 from typing import AsyncGenerator
 
+import aioredis
 from aiohttp import ClientSession, web
 from apscheduler.schedulers.asyncio import AsyncIOScheduler
 from slack.io.aiohttp import SlackAPI
 
 from . import tasks
 
 
+async def apscheduler(app: web.Application) -> AsyncGenerator[None, None]:
+    app["scheduler"] = scheduler = AsyncIOScheduler()
+    scheduler.start()
+
+    yield
+
+    scheduler.shutdown()
+
+
 async def background_jobs(app: web.Application) -> AsyncGenerator[None, None]:
     scheduler = app["scheduler"]
 
     scheduler.add_job(
-        tasks.sync_github_repositories(app),
+        tasks.sync_github_repositories,
         "cron",
         minute=30,
         jitter=30,
         next_run_time=datetime.utcnow() + timedelta(seconds=5),
```

Comment:
We can probably get rid of that. There is no use to sync at startup since the data is in redis (same for the two other tasks).

Reply:
I'm going to leave it for a release to populate the cache and do a follow-up to remove them, does that work okay for you?